### PR TITLE
Feature/gidd good practice

### PR DESCRIPTION
**Pull request by Rup-Narayan-Rajbanshi**
- Addresses https://github.com/idmc-labs/idmc-website-components/issues/44

## Changes
- Make required field optional, add fields, rename fields, 

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] n+1 queries
- [x] flake8 issues
- [x] `print`
- [x] typos
- [x] unwanted comments

## This PR contains valid:

- [ ] tests
- [ ] permission checks (tests here too)
- [x] translations

### CODE REVIEW

**Review by thenav56 — CHANGES_REQUESTED** [review]

File: apps/good_practice/models.py [hunk 1]
```diff
@@ -170,7 +175,7 @@ class StageType(models.TextChoices):
         null=True,
         max_length=2000,
     )
-    under_review = models.BooleanField(default=True)
+    under_review = models.BooleanField(default=False)
```
Comment (thenav56) [inline apps/good_practice/models.py:178]: Let's use the migration runPython to set False for existing entries. For new ones let's use default=True and don't let client assign this value.

File: apps/good_practice/mutations.py [hunk 2]
```diff
@@ -24,8 +29,11 @@ class GoodPracticeInputType:
     captcha: str
 
     # Enum fields
-    type: Optional[str]
-    stage: Optional[str]
+    # type: Optional[str]
+    # stage: Optional[str]
```
Comment (thenav56) [inline apps/good_practice/mutations.py:33]:
```suggestion
```

File: apps/good_practice/tests/test_mutations.py [hunk 3]
```diff
@@ -138,6 +140,7 @@ def test_create_goodpractice(self):
             response = self.query_check(
                 create_mutation, variables={"input": input_variable}
             )
+            print(response)
```
Comment (thenav56) [inline apps/good_practice/tests/test_mutations.py:143]:
```suggestion
```

**Review by thenav56 — CHANGES_REQUESTED** [review]

File: apps/good_practice/models.py (hunk 1 above)
Comment (bimal125) [reply apps/good_practice/models.py]: @thenav56  changed logic accordingly

**Review by bimal125 — APPROVED** [review]

**Review by thenav56 — APPROVED** [review]
It's okay to merge now. But let's do this in future PR

- Let's try to use Enum instead. Maybe define a default value here instead. https://github.com/idmc-labs/gidd-server/pull/28#discussion_r1122851026
- Rename `createGoodPractice` -> `publicCreateGoodPractice`

**Pull request merged**